README.md

```
@@ -92,6 +92,15 @@ The SDK is also available on the [Unity Asset Store](https://assetstore.unity.co
 6. Create new Canvas
 7. Import WalletHolder prefab into the Canvas or if you want your design just import wallet prefab and customize the scene.
 
+## 📱 Android Build Setup
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)